test/integration/test_integration_basics.py

```diff
@@ -269,6 +269,32 @@ def test_https_h1_tls_context_configuration(https_test_server_fixture):
   assertCounterEqual(counters, "ssl.ciphers.ECDHE-RSA-CHACHA20-POLY1305", 1)
 
 
+def test_https_h1_transport_socket_configuration(https_test_server_fixture):
```

Comment:
- possibly we can refactor this a little bit to share code with the existing `test_XXX_tls_context_configuration` tests, as only the arguments to `runNighthawkClient()` change between those?
- also a tiny nit, it's better to make separate elements for arg / value pairs, e.g. `""--transport-socket ", "{..}"`. This makes it easier to inspect/massage the arguments in `runNighthawkClient()`. (There's may be some existing inconsistencies with respect to this).

Reply:
Done and done!